app/controllers/cart_items_controller.rb

```diff
@@ -0,0 +1,34 @@
+class CartItemsController < ApplicationController
+  def index
+    if logged_in?
+      @cart_items = current_user.cart.cart_items
+    end
+  end
+
+  def new
+    @cart_item = CartItem.new
+  end
+
+  def show
+    @dish = Dish.find(params[:id])
+  end
+
+  def create
+    @cart_item = CartItem.new(cart_item_params)
```

Comment:
A better approach would be to avoid sending the `cart_id` from the front-end. Could remove it from the `cart_item_params` and simply replace it with `current_user.cart.id`